cycode/cli/commands/scan/code_scanner.py

```diff
@@ -148,14 +148,14 @@ def _enrich_scan_result_with_data_from_detection_rules(
 
 def _get_scan_documents_thread_func(
     context: click.Context, is_git_diff: bool, is_commit_range: bool, scan_parameters: dict
-) -> Callable[[List[Document]], Tuple[str, CliError, LocalScanResult]]:
+) -> tuple[Callable[[list[Document]], tuple[str, CliError, LocalScanResult]], str]:
```

Comment:
pls return back List and Tuple. it will not work for all Python versions which we support